todoman/cli.py

```diff
@@ -143,6 +143,11 @@ def cli(ctx, color, porcelain):
     if len(paths) == 0:
         click.echo("No lists found.\n")
         click.echo(config["main"]["path"])
+        click.echo("goto ~/.local/share/calendars")
```

Comment:
`~/.local/share/calendars` is the default path, but the user may have configured another one. This is what `config["main"]["path"]` is for.

Something like: `"No lists found matching {}, create a directory for a new list".format(config["main"]["path"]`